cypress-slack-reporter-old/src/slack/slack-alert.ts

```diff
@@ -62,15 +63,16 @@ interface ReportStatistics {
 }
 
 export const slackRunner = async ({
-  ciProvider,
```

Comment:
It would be good to revert any whitespace changes, as it makes it difficult for a reviewer to determine what has changed, and what hasn't :)